srml/generic-asset/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "srml-generic-asset"
+version = "2.0.0"
+authors = ["Centrality Developers <support@centrality.ai>"]
+edition = "2018"
+
+[dependencies]
+serde = { version = "1.0", default-features = false }
```

Comment:
Instead of disabling the default features, we should make this an optional depedency.